.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -22,8 +22,16 @@ TODO: Describe your changes in detail here.
 <!--- Using the above wording causes Github to automatically close the issue on merge. -->
 Closes #ISSUE_NUMBER.
 
-## Verification Steps
-<!-- Please describe in detail how a reviewer can verify your changes. -->
+## Acceptance 
+<!-- The people and processes this pull request needs before it is merged. -->
+<!-- These fields are not required when opening the pull request, but they -->
+<!-- should be populated after code review. -->
+### Verification Stakeholders
+<!-- People who must verify that this solves the attached issue. -->
+
```

Comment:
@zetlen Can we add, if PR impacts upward-php area yes/no? It will help me in testing :) .